exotations/dynamics_solvers/exotica_pinocchio_dynamics_solver/src/pinocchio_dynamics_solver.cpp

```diff
@@ -80,4 +80,25 @@ Eigen::MatrixXd PinocchioDynamicsSolver::fu(const StateVector& x, const ControlV
     // return B_;
 }
 
+Eigen::Tensor<double, 3> PinocchioDynamicsSolver::fxx(const StateVector& x, const ControlVector& u)
```

Comment:
Add a TODO comment for the whole class or maybe even throw an 'NotYetImplemented' exception when used.